packages/lesswrong/components/tagging/TagSubforumPage2.tsx

```diff
@@ -548,22 +551,24 @@ const TagSubforumPage2 = ({classes}: {
         <div className={classes.feedHeaderButtons}>
           <Link to={`/newPost?subforumTagId=${tag._id}`} className={classes.newPostLink}>
             <SectionButton>
-              <AddBoxIcon /> New Post
+              <AddBoxIcon /> <span className={classes.hideOnMobile}>New</span> Post
             </SectionButton>
           </Link>
           <SectionButton onClick={clickNewDiscussion}>
-            <AddBoxIcon /> New Discussion
+            <AddBoxIcon /> <span className={classes.hideOnMobile}>New</span> Discussion
```

Comment:
This is to squeeze in these two buttons and the sorting dropdown without it wrapping in most cases